packages/@aws-cdk/aws-scheduler-alpha/test/input.test.ts

```diff
@@ -75,12 +80,11 @@ describe('schedule target input', () => {
   test('create an input from object with a ref', () => {
     new Schedule(stack, 'MyScheduleDummy', {
       schedule: expr,
-      target: new targets.LambdaInvoke({
-        role,
+      target: new targets.LambdaInvoke(func, {
```

Comment:
just confirming these diffs are not breaking changes because `targets.LambdaInvoke` was never exposed.